.github/workflows/validate.yml

```diff
@@ -0,0 +1,33 @@
+name: ValidateTrialJsonSchema
+on:
+  push:
+    branches: [ main, feature/json-validation ]
```

Comment:
need to delete the 'feature/json-validation'

Reply:
Done